Model/InstantPurchase/AbstractAdyenTokenFormatter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2022 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Model\InstantPurchase;
+
+use Magento\Vault\Api\Data\PaymentTokenFactoryInterface;
+use Magento\Vault\Api\Data\PaymentTokenInterface;
+
+abstract class AbstractAdyenTokenFormatter
+{
+    /**
+     * Most used credit card types
+     *
+     * @var array
+     */
+    public static $baseCardTypes = [
```

Comment:
Should we include `KCP` into this list?

Reply:
We can add it in the future but for now I'd rather leave it as the same array that it currently is.